### ISSUE TITLE
Navbar dropdown-with-icon anchor with long text overflow

### ISSUE THREAD

**Issue by AzrielOmega** [timcreative/get-shit-done-pro#25]
Anchor with long text (18 characters or more) will flow out of the container. And with "whitespace: nowrap" on mobile some time the text will flow out of the screen.

**Comment by AzrielOmega** [comment]
Issue come from float: left; in

.dropdown-with-icons i {
  text-align: center;
  line-height: 28px;
  display: left;
}

**Comment by AzrielOmega** [comment]
Solution on desktop device:
Step 1: Seperate text in anchor to a span. Example: 
<li>
<a href="#">
<i class="pe-7s-ticket"></i>
<span> This is a really long text for demostration</span>
</a>
</li>

Step 2: Add CSS:
.dropdown-with-icons span {
  display:inline-block;
  vertical-align:top;
}

**Comment by AzrielOmega** [comment]
Issue #2: On mobile device the content in <span> will overflow out of screen.

Solution: Modify CSS:
```
.navbar-nav .open .dropdown-menu > li > a {
    padding: 15px 15px 5px 20px; ***
    margin-right: 50px;
  }
```
*** I reduced padding on left side to 20px for better look.

**Comment by alexandru-paduraru** [comment]
Thank you for pointing out and also thank you for the quick fixes. We will check these issues and test some usecases with your fix. If that is working fine we will add it in the next v1.5 updated.

Best,
Alex

**Comment by alexandru-paduraru** [comment]
Fixed in v1.4.2 using another trick with position absolute for icons, so you don't have to write again everything in a span. Please check!
